CONTRIBUTING.rst

```diff
@@ -140,13 +140,12 @@ in the dask-image repository.
 There is a separate environment file for each supported Python version.
 
 We will use conda to
-`create an environment from a file
-<https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file>`_
+`create an environment from a file`_
```

Comment:
Spotted this long in text URL. Thought it would be better to extract this. So moved it below with the 2 other URLs needed for text update